### PR TITLE
 IDVA5-674-Refactoring-controllers

### PR DESCRIPTION
**Pull request by abhiahirv1**
Refactored the code for clarity and consistency.
Properly aligned imports for better readability.
Added JSDoc comments for documentation purposes.
Ensured consistent use of whitespace and indentation.

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/controllers/features/limited/companyLookupController.ts [hunk 1]
```diff
@@ -2,18 +2,16 @@ import { Session } from "@companieshouse/node-session-handler";
...
 import { ValidationError, validationResult } from "express-validator";
 import * as config from "../../../config";
-import { CompanyDetailsService } from "../../../services/company-details/companyDetailsService";
 import { selectLang, addLangToUrl, getLocalesService, getLocaleInfo } from "../../../utils/localise";
 import { BASE_URL, LIMITED_IS_THIS_YOUR_COMPANY, LIMITED_WHAT_IS_THE_COMPANY_NUMBER, TYPE_OF_BUSINESS } from "../../../types/pageURL";
-import { CompanyLookupService } from "../../../services/companyLookupService";
-import { FormattedValidationErrors, formatValidationError } from "../../../validation/validation";
-
-const companyDetailsService = new CompanyDetailsService();
+import { CompanyLookupService, getCompanyDetails } from "../../../services/companyLookupService";
```
Comment (sampankumar) [inline src/main/controllers/features/limited/companyLookupController.ts:7]: Please check if certain imports are still required

File: src/main/controllers/features/limited/isThisYourCompanyController.ts [hunk 2]
```diff
@@ -23,14 +26,16 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
...
+
         const session: Session = req.session as any as Session;
         const company : Company = session?.getExtraData(COMPANY_DETAILS)!;
-        if (company.status === "active") {
-            res.redirect(addLangToUrl(BASE_URL + LIMITED_WHAT_IS_YOUR_ROLE, lang));
-        } else {
-            res.redirect(addLangToUrl(BASE_URL + LIMITED_COMPANY_INACTIVE, lang));
-        }
+
+        const redirectUrlAccordingToRole = company.status === "active"
```
Comment (sampankumar) [inline src/main/controllers/features/limited/isThisYourCompanyController.ts:35]: May be personal opinion I find the above if condition more readable.

File: src/main/controllers/features/limited/nameRegisteredWithAmlController.ts [hunk 3]
```diff
@@ -32,19 +37,13 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
...
             });
         } else {
-            const nextPageUrl = addLangToUrl(BASE_URL + LIMITED_SECTOR_YOU_WORK_IN, lang);
-            const nextPageUrlForBoth = addLangToUrl(BASE_URL + LIMITED_BUSINESS_MUSTBE_AML_REGISTERED_KICKOUT, lang);
-            if (selectedOption === "YOUR_NAME") {
-                res.redirect(nextPageUrlForBoth); // Redirect to another page when your name selected
-            } else {
-                res.redirect(nextPageUrl); // Redirect to the sector page for the other 2 options
-            }
+            const redirectUrlAccordingToRole = selectedOption === "YOUR_NAME"
```
Comment (sampankumar) [inline src/main/controllers/features/limited/nameRegisteredWithAmlController.ts:40]: same as above

Comment (abhiahirv1) [reply src/main/controllers/features/limited/nameRegisteredWithAmlController.ts]: Done

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by ttingle-ch — APPROVED** [review]

**Review by swalke-ch — APPROVED** [review]

**Pull request merged**